scripts/lang/en/form_of.py

```diff
@@ -0,0 +1,668 @@
+"""
+List of form-of templates https://en.wiktionary.org/wiki/Category:Form-of_templates
+From: https://en.wiktionary.org/wiki/Template:place
+Current version:
+    2020-12-02T14:47:01
+    https://en.wiktionary.org/w/index.php?title=Category:Form-of_templates&oldid=60765059
+List generated by
```

Comment:
Let's add the Wiki page, right? :)

Reply:
Yes, I wanted your input on the approach before creating the wiki page